### PR TITLE
add rule proc_creation_lnx_esxcli_system_enumeration

### PR DESCRIPTION
**Pull request by kidrek**
### Summary of the Pull Request

This pull request implements a new rule to detect Vmware ESXi system enumeration.

### Example Log Event

```
<14>2023-08-22T15:48:21Z ESXi.home shell[3707386]: [root]: esxcli system version get
<14>2023-08-22T15:48:21Z ESXi.home shell[3707386]: [root]: esxcli system account list
```

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Comment by kidrek** [comment]
Thx @nasbench.
In fact, these other rules were also ready on my side. I was waiting for this first ESXi rule to be validated before submitting the others ;).

**Comment by nasbench** [comment]
> Thx @nasbench. In fact, these other rules were also ready on my side. I was waiting for this first ESXi rule to be validated before submitting the others ;).

Nice. Well I added you as author then :) 

Cheers mate

File: rules/linux/process_creation/proc_creation_lnx_esxcli_network_discovery.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+title: ESXi Network Configuration Discovery Via Esxcli
+id: 33e814e0-1f00-4e43-9c34-31fb7ae2b174
+status: experimental
+description: Detects execution of "esxcli" command with the "network" flag in order to retrieve information about the network configuration.
```
Comment (phantinuss) [inline rules/linux/process_creation/proc_creation_lnx_esxcli_network_discovery.yml:4]:
```suggestion
description: Detects execution of the "esxcli" command with the "network" flag in order to retrieve information about the network configuration.
```

File: rules/linux/process_creation/proc_creation_lnx_esxcli_permission_change_admin.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+title: ESXi Admin Permission Assigned To Account Via Esxcli
```
Comment (phantinuss) [inline rules/linux/process_creation/proc_creation_lnx_esxcli_permission_change_admin.yml:1]:
```suggestion
title: ESXi Admin Permission Assigned To Account Via ESXCLI
```

File: rules/linux/process_creation/proc_creation_lnx_esxcli_permission_change_admin.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+id: 9691f58d-92c1-4416-8bf3-2edd753ec9cf
+status: experimental
+description: Detects execution of "esxcli" command with the "system" and "permission" flags in order to assign admin permissions to an account.
```
Comment (phantinuss) [inline rules/linux/process_creation/proc_creation_lnx_esxcli_permission_change_admin.yml:4]:
```suggestion
description: Detects execution of the "esxcli" command with the "system" and "permission" flags in order to assign admin permissions to an account.
```

File: rules/linux/process_creation/proc_creation_lnx_esxcli_system_discovery.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+title: ESXi System Information Discovery Via Esxcli
```
Comment (phantinuss) [inline rules/linux/process_creation/proc_creation_lnx_esxcli_system_discovery.yml:1]:
```suggestion
title: ESXi System Information Discovery Via ESXCLI
```

File: rules/linux/process_creation/proc_creation_lnx_esxcli_system_discovery.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+id: e80273e1-9faf-40bc-bd85-dbaff104c4e9
+status: experimental
+description: Detects execution of "esxcli" command with the "system" flag in order to retrieve information about the different component of the system. Such as accounts, modules, ntp, etc.
```
Comment (phantinuss) [inline rules/linux/process_creation/proc_creation_lnx_esxcli_system_discovery.yml:4]:
```suggestion
description: Detects execution of the "esxcli" command with the "system" flag in order to retrieve information about the different component of the system. Such as accounts, modules, NTP, etc.
```

File: rules/linux/process_creation/proc_creation_lnx_esxcli_vm_discovery.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
+title: ESXi VM List Discovery Via Esxcli
```
Comment (phantinuss) [inline rules/linux/process_creation/proc_creation_lnx_esxcli_vm_discovery.yml:1]:
```suggestion
title: ESXi VM List Discovery Via ESXCLI
```

File: rules/linux/process_creation/proc_creation_lnx_esxcli_vm_discovery.yml [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+id: 5f1573a7-363b-4114-9208-ad7a61de46eb
+status: experimental
+description: Detects execution of "esxcli" command with the "vm" flag in order to retrieve information about the installed VMs.
```
Comment (phantinuss) [inline rules/linux/process_creation/proc_creation_lnx_esxcli_vm_discovery.yml:4]:
```suggestion
description: Detects execution of the "esxcli" command with the "vm" flag in order to retrieve information about the installed VMs.
```

File: rules/linux/process_creation/proc_creation_lnx_esxcli_vm_kill.yml [hunk 8]
```diff
@@ -0,0 +1,26 @@
+title: ESXi VM Kill Via Esxcli
```
Comment (phantinuss) [inline rules/linux/process_creation/proc_creation_lnx_esxcli_vm_kill.yml:1]:
```suggestion
title: ESXi VM Kill Via ESXCLI
```

File: rules/linux/process_creation/proc_creation_lnx_esxcli_vm_kill.yml [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+id: 2992ac4d-31e9-4325-99f2-b18a73221bb2
+status: experimental
+description: Detects execution of "esxcli" command with the "vm" and "kill" flag in order to kill/shutdown a specific VM.
```
Comment (phantinuss) [inline rules/linux/process_creation/proc_creation_lnx_esxcli_vm_kill.yml:4]:
```suggestion
description: Detects execution of the "esxcli" command with the "vm" and "kill" flag in order to kill/shutdown a specific VM.
```

File: rules/linux/process_creation/proc_creation_lnx_esxcli_vsan_discovery.yml [hunk 10]
```diff
@@ -0,0 +1,30 @@
+title: ESXi Vsan Information Discovery Via Esxcli
```
Comment (phantinuss) [inline rules/linux/process_creation/proc_creation_lnx_esxcli_vsan_discovery.yml:1]:
```suggestion
title: ESXi VSAN Information Discovery Via ESXCLI
```

File: rules/linux/process_creation/proc_creation_lnx_esxcli_vsan_discovery.yml [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+id: d54c2f06-aca9-4e2b-81c9-5317858f4b79
+status: experimental
+description: Detects execution of "esxcli" command with the "vsan" flag in order to retrieve information about virtual storage. Seen used by malware such as DarkSide
```
Comment (phantinuss) [inline rules/linux/process_creation/proc_creation_lnx_esxcli_vsan_discovery.yml:4]:
```suggestion
description: Detects execution of the "esxcli" command with the "vsan" flag in order to retrieve information about virtual storage. Seen used by malware such as DarkSide.
```

**Review by phantinuss — APPROVED** [review]

**Pull request merged**